src/core_plugins/status_page/public/components/metric_tiles.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import formatNumber from '../lib/format_number';
+import React, { Component } from 'react';
+import { Metric as MetricPropType } from '../lib/prop_types';
+import PropTypes from 'prop-types';
+import {
+  EuiFlexGrid,
+  EuiFlexItem,
+  EuiCard,
+} from '@elastic/eui';
+
+
+/*
+Displays a metric with the correct format.
+*/
+export class MetricTile extends Component {
+  static propTypes = {
+    metric: MetricPropType.isRequired
+  };
+
+  formattedMetric() {
+    const { value, type } = this.props.metric;
+
+    const metrics = [].concat(value);
+    return metrics.map(function (metric) {
+      return formatNumber(metric, type);
+    }).join(', ');
+  }
+
+  render() {
+    const { name } = this.props.metric;
+
+    return (
+      <EuiCard
+        layout="horizontal"
+        title={this.formattedMetric()}
+        description={name}
+      />
+    );
+  }
+}
+
+/*
+Wrapper component that simply maps each metric to MetricTile inside a FlexGroup
+*/
+const MetricTiles = ({
+  metrics
+}) => (
+  <EuiFlexGrid columns={3}>
+    {
+      metrics.map(metric => (
+        <EuiFlexItem key={metric.name}>
+          <MetricTile metric={metric} />
+        </EuiFlexItem>
+      ))
+    }
+  </EuiFlexGrid>
+);
+
+MetricTiles.propTypes = {
+  metrics: PropTypes.arrayOf(MetricPropType).isRequired
+};
+
+export default MetricTiles;
```

Comment:
nit:  styleguide convention for no default exports, guess our ci task to lint didn't catch it?

Reply:
Hmm yeah it's passing lint. We have almost 1k `export default`s, but happy to change it if this is the direction we want to move to.

Reply:
Good catch @jbudz - let's go ahead and update this to reflect https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#use-named-exports-only

Reply:
@spalger any idea why this wasn't catch by the linter?

Reply:
That rule actually looks like it's _allowing_ default exports, not preventing.

Reply:
Probably need to cut a new issue for changing all the existing ones. I'll make sure this PR is good though 👍